### PR TITLE
Simplify data dir management

### PR DESCRIPTION
**Pull request by hackdna**

### CODE REVIEW

**Comment by hackdna** [comment]
It's very hard to parse the test output and tests don't always run reliably, so I thought those failures were either not important or transient. However, the site appears to be functional both in Vagrant and on AWS.
I've tried to run the test suite for `tool_manager` locally to try and debug the problem but it doesn't complete (the same symptoms as #2067).

**Comment by scottx611x** [comment]
@hackdna If you are unable to test locally why not edit the tests to yield some more useful information before the [failing assertion](https://github.com/refinery-platform/refinery-platform/blob/80f4c67c72bdc2345bbc4caeab4755059cfe8609/refinery/tool_manager/tests.py#L2731) then push to travis again?

**Comment by hackdna** [comment]
Or perhaps we could fix the tests so they do run locally as intended? :)

**Comment by scottx611x** [comment]
Sorry ^^^ misclicked

**Comment by scottx611x** [comment]
@hackdna Sure, I think thats within the scope of #2067. These 400s you're seeing don't seem related to #2067.

**Comment by scottx611x** [comment]
@hackdna If you're most recent build here fails, I believe the fix would be to add the default like: 
`"DJANGO_DOCKER_ENGINE_DATA_DIR": "<%= @django_docker_engine_data_dir || "/data" %>",`
in config.json.erb

**Comment by hackdna** [comment]
@scottx611x @mccalluc This PR is now passing the test suite. Please review and merge at will (note that it's based on develop now, not on release-1.6.0 branch).

**Comment by hackdna** [comment]
This PR was originally based on release-1.6.0 branch, so it contain some of those commits. The release branch will be merged back into develop eventually, so the end result will be the same.

File: deployment/manifests/default.pp [hunk 1]
```diff
@@ -8,29 +8,19 @@
 $isa_tab_dir = "${project_root}/isa-tab"
 $media_root = "${project_root}/media"
 $import_dir = "${project_root}/import"
-$solr_custom_synonyms_file =
-  "${django_root}/solr/core/conf/custom-synonyms.txt"
+$data_dir = '/data'
+$docker_dir = "${data_dir}/docker"
+$solr_dir = "${data_dir}/solr"
+$solr_custom_synonyms_file = "${django_root}/solr/core/conf/custom-synonyms.txt"
```
Comment (mccalluc) [inline deployment/manifests/default.pp]: django_root -> data_dir?

**Comment by mccalluc** [comment]
@hackdna : Not sure where this is going but I'm going to make a branch off of this, try to resolve the conflicts, and incorporate the one typo fix... If all of that is straight forward and travis is happy, I'll go ahead and close this.

**Comment by hackdna** [comment]
I've left this one on the back burner. It probably needs more work now than just resolving merge conflicts because Docker data dir has moved to the root volume on AWS.

**Comment by hackdna** [comment]
No longer applicable.
